src/styles/_catalog.scss

```diff
@@ -1,22 +1,17 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(auto-fill,minmax(200px, 1fr));
```

Comment:
![image](https://user-images.githubusercontent.com/86923089/209963041-8080eb8a-550a-4538-b256-b06758414cae.png)

It always should be 48px (use media queries)
![image](https://user-images.githubusercontent.com/86923089/209963063-03d9c559-ebc9-46bc-87b9-8db242cfd3ea.png)